src/Data/customerData.js

```diff
@@ -0,0 +1,22 @@
+import Parse from "parse";
+
+const Customer = Parse.Object.extend("Customer");
+
+export default async function getAllCustomers() {
+
+    const allCustomersQuery = new Parse.Query(Customer);
+    const allCustomers = await allCustomersQuery.find();
```

Comment:
I know your code is in sync with the lecture slides, but question to consider: What happens if find() fails? Or if the returned array is empty? Are you handling this?